dot-net-sdk/http/ConfigurationRequester.cs

```diff
@@ -52,10 +52,10 @@ public void LoadConfiguration()
         {
             // Fetch methods throw if resource is null.
             var flags = flagConfigurationResponse.Resource!;
-            var banditFlags = flags.Bandits ?? new BanditFlags();
+            var indexer = flags.BanditReferences ?? new BanditReferences();
 
             IEnumerable<Bandit> banditModelList = Array.Empty<Bandit>();
-            if (banditFlags.Count > 0)
+            if (indexer.HasBanditReferences())
```

Comment:
Planning to only fetch if we don't have the model versions loaded in a subsequent PR?